doc-content/jbpm-docs/src/main/asciidoc/Kafka/kieserver-kafka-emit-proc.adoc

```diff
@@ -51,7 +51,26 @@ endif::JBPM,DROOLS,OP[]
 . Configure any of the following system properties for the {KIE_SERVER} as necessary:
 * `org.kie.jbpm.event.emitters.kafka.bootstrap.servers`: The host and port of the Kafka broker. The default value is `localhost:9092`. You can use a comma-separated list of multiple host:port pairs.
 * `org.kie.jbpm.event.emitters.kafka.date_format`: The timestamp format for the `time` field of the messages. The default value is `yyyy-MM-dd'T'HH:mm:ss.SSSZ` .
-* `org.kie.jbpm.event.emitters.kafka.topic.processes`: The topic name for process event messages. The default value is `jbpm-processes-events`. 
-* `org.kie.jbpm.event.emitters.kafka.topic.cases`: The topic name for process event messages. The default value is `jbpm-cases-events`. 
-* `org.kie.jbpm.event.emitters.kafka.topic.tasks`: The topic name for process event messages. The default value is `jbpm-processes-tasks`. 
+* `org.kie.jbpm.event.emitters.kafka.topic.processes`: The topic name for process event messages. The default value is `jbpm-processes-events`.
+* `org.kie.jbpm.event.emitters.kafka.topic.cases`: The topic name for process event messages. The default value is `jbpm-cases-events`.
+* `org.kie.jbpm.event.emitters.kafka.topic.tasks`: The topic name for process event messages. The default value is `jbpm-processes-tasks`.
 * `org.kie.jbpm.event.emitters.kafka.client.id`: An identifier string to pass to the server when making requests. The server uses this string for logging.
+* `org.kie.jbpm.event.emitters.kafka._property_name_`: You can set any {KAFKA_PRODUCT} consumer or producer property by using the `org.kie.jbpm.event.emitters.kafka` prefix. For example, to set a value for the `buffer.memory` producer property, set the `org.kie.jbpm.event.emitters.kafka.buffer.memory` {KIE_SERVER} system property.
++
+This setting applies when the {KIE_SERVER} emits Kafka messages automatically when completing transactions.
++
+For a list of {KAFKA_PRODUCT} consumer properties, see
```

Comment:
This is OK, but can you combine the two? 
For lists of {KAFKA_PRODUCT} consumer and producer properties, see...

Since they are both in the same doc?

Reply:
They are different chapters in teh same doc, we probably want to discuss the formatting?